support-frontend/assets/stylesheets/gu-sass/typography.scss

```diff
@@ -8,9 +8,9 @@
 // ----- Font Families ----- //
 
 $gu-titlepiece: 'GT Guardian Titlepiece', Georgia, serif !default;
-$gu-headline: 'Guardian Egyptian Web', Georgia, serif !default;
-$gu-text-egyptian-web: 'Guardian Text Egyptian Web', Georgia, serif !default;
-$gu-text-sans-web: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif !default;
+$gu-headline: 'GH Guardian Headline', Georgia, serif !default;
+$gu-text-egyptian-web: 'GuardianTextEgyptian', Georgia, serif !default;
+$gu-text-sans-web: 'GuardianTextSans', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif !default;
```

Comment:
How would we replicate this in Emotion? (or is it handled by Source?) Sass is a legacy part of the codebase, it would be helpful to know how we migrate away from this in future.

Reply:
@i-hardy I believe the typography exports from Source map to font-families that reference the newer font names as the primary font and use the old font name as a fallback if the new font in unavailable.

Reply:
Here's a link to where Source sets the font values which backs up what George has said above: https://github.com/guardian/source/blob/main/src/core/foundations/src/theme.ts#L27

You can then use the Typography methods that it provides:
https://theguardian.design/2a1e5182b/p/95d5d0-code

Reply:
these are [the font family name source uses](https://github.com/guardian/source/blob/0566752b3ce5851bc1a907db017e74819fcdcec1/src/core/foundations/src/theme.ts#L27-L35), so in theory you shouldn't need to do anything if you're using source.

cc @SiAdcock to make sure i've got that right!